### PR TITLE
IBX-7501: Editing embedded content items described in User Documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-7501)
| Versions      | master
| Edition       | Connect, Experience, Commerce

Editing embedded content items described in User Documentation.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

**Review by lucasOsti — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

File: docs/content_management/create_edit_content_items.md [hunk 1]
```diff
@@ -120,6 +120,27 @@ To add a new element to the Field, choose one of the available elements:
...
+### Edit embedded Content items
+
+You can edit embedded Content items without leaving current window.
+
+To do it, first insert selected Content item in the Rich Text Field.
+Then, click the icon on the right side and click **Edit**.
+
+![Edit embedded Content item](img/edit_embedded_item_richtext.png "Edit embedded Content item")
+
+This action opens a new tab in the browser with an editing screen of the selected Content item.
```
Comment (dabrt) [inline docs/content_management/create_edit_content_items.md:132]:
```suggestion
This action opens a new browser tab with an editing screen of the selected Content item.
```

File: docs/content_management/create_edit_content_items.md [hunk 2]
```diff
@@ -120,6 +120,27 @@ To add a new element to the Field, choose one of the available elements:
...
+When you finish editing the item, click **Publish**.
```
Comment (dabrt) [inline docs/content_management/create_edit_content_items.md:133]:
```suggestion
When you are done editing the item, click **Publish**.
```

File: docs/content_management/create_edit_content_items.md [hunk 3]
```diff
@@ -120,6 +120,27 @@ To add a new element to the Field, choose one of the available elements:
...
+To see implemeneted changes refresh the browser page.
```
Comment (dabrt) [inline docs/content_management/create_edit_content_items.md:134]:
```suggestion
You can now close the additional browser tab and refresh the original editing view to see the changes.
```

File: docs/content_management/create_edit_content_items.md [hunk 4]
```diff
@@ -120,6 +120,27 @@ To add a new element to the Field, choose one of the available elements:
...
+
+If the Content item has more than one translation available, you need to select the language.
+
```
Comment (dabrt) [inline docs/content_management/create_edit_content_items.md:137]:
```suggestion
```

File: docs/content_management/create_edit_content_items.md [hunk 5]
```diff
@@ -120,6 +120,27 @@ To add a new element to the Field, choose one of the available elements:
...
+![Edit embedded Content item - select language](img/edit_embedded_item_language_richtext.png "Edit embedded Content item - select language")
+
+This option is also available when you want to set up a [relation](../configure_ct_field_settings/#content-relation-settings) with another Content item.
```
Comment (dabrt) [inline docs/content_management/create_edit_content_items.md:140]:
```suggestion
!!! tip

	Editing another Content item in an external window is also available when you are setting a [relation](../configure_ct_field_settings/#content-relation-settings) with another Content item.
```

File: docs/content_management/create_edit_content_items.md (hunk 3 above)
Comment (juskora) [inline docs/content_management/create_edit_content_items.md:134]:
```suggestion
To see implemented changes refresh the browser page.
```

**Review by juskora — APPROVED** [review]

File: docs/content_management/create_edit_content_items.md (hunk 5 above)
Comment (julitafalcondusza) [reply docs/content_management/create_edit_content_items.md:140]: I'd keep it as a sentence, not as a tip.

File: docs/content_management/create_edit_content_items.md (hunk 3 above)
Comment (julitafalcondusza) [reply docs/content_management/create_edit_content_items.md:134]: In my opinion, the shorter, the better.

File: docs/content_management/create_edit_content_items.md (hunk 2 above)
Comment (juskora) [reply docs/content_management/create_edit_content_items.md:133]: Let's avoid passive voice.

File: docs/content_management/create_edit_content_items.md [hunk 6]
```diff
@@ -120,6 +120,27 @@ To add a new element to the Field, choose one of the available elements:
 Each of these elements can have its own settings, such as text formatting.
 The option bar also lets you reorder or remove any elements in the Rich Text Field.
 
+### Edit embedded Content items
+
+You can edit embedded Content items without leaving current window.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/create_edit_content_items.md:125]: Yes, agree. "Edit embedded Content items" section is inside of the "Edit Rich Text Fields" section, so there is no need to repeat "(...) within the Rich Text Field (...)". Original sentence is shorter and provides exactly the same information.

[the remark this answers is not in the document]
Comment (dabrt) [reply docs/content_management/create_edit_content_items.md:125]: You are right, originally it was not inside Edit RTF" section, hence the comment.

**Review by dabrt — APPROVED** [review]

**Pull request merged**